exchanges/bitmex/bitmex_test.go

```diff
@@ -59,6 +60,14 @@ func TestStart(t *testing.T) {
 	testWg.Wait()
 }
 
+func TestGetFullFundingHistory(t *testing.T) {
+	_, err := b.GetFullFundingHistory("", "", "", "", "", true, time.Time{}, time.Time{})
```

Comment:
It would be good to test an additional scenario where these params are filled in